docs/docusaurus/docs/guides/expectations/how_to_create_and_edit_expectations_based_on_domain_knowledge_without_inspecting_data_directly.md

```diff
@@ -20,109 +23,129 @@ The following are the reasons why you might want to do this:
+- Created a Datasource from which to request a Batch of data for introspection
 
 </Prerequisites>
 
+<details>
+<summary>
+
+### If you haven't set up Great Expectations
+
+</summary>
+
+<IfYouStillNeedToSetupGX />
+
+</details>
+
+<details>
+<summary>
+
+### If you haven't initialized your Data Context
+
+</summary>
+
+See one of the following guides:
+
+<DataContextInitializeQuickOrFilesystem />
+
+</details>
+
+<details>
+<summary>
+
+### If you haven't created a Datasource
+
+</summary>
+
+See one of the following guides:
+
+<ConnectingToDataFluently />
+
+</details>
+
 ## Steps
 
-### 1. Use the CLI to generate a helper notebook
+### 1. Import the Great Expectations module and instantiate a Data Context
+
+For this guide we will be working with Python code in a Jupyter Notebook. Jupyter is included with GX and lets us easily edit code and immediately see the results of our changes.
 
-From the command line, use the <TechnicalTag tag="cli" text="CLI" /> to run:
+Run the following code to import Great Expectations and instantiate a Data Context:
 
-```bash
-great_expectations suite new
+```python name="tests/integration/docusaurus/expectations/how_to_create_and_edit_an_expectationsuite_domain_knowledge.py get_data_context"
 ```
 
-### 2. Create Expectation Configurations in the helper notebook
+:::info Data Contexts and persisting data
+
+If you're using an Ephemeral Data Context, your configurations will not persist beyond the current Python session.  However, if you're using a Filesystem or Cloud Data Context, they do persist.  The `get_context()` method returns the first Cloud or Filesystem Data Context it can find.  If a Cloud or Filesystem Data Context has not be configured or cannot be found, it provides an Ephemeral Data Context.  For more information about the `get_context()` method, see [How to quickly instantiate a Data Context](/docs/guides/setup/configuring_data_contexts/instantiating_data_contexts/how_to_quickly_instantiate_a_data_context).
+
+:::
+
+### 2. Use an existing Data Asset to create a Batch Request
+
+Add the following method to retrieve a previously configured Data Asset from the Data Context you initialized and create a Batch Request to identify the Batch of data that you'll use to validate your Expectations:
+
+```python name="tests/integration/docusaurus/expectations/how_to_create_and_edit_an_expectationsuite_domain_knowledge.py get_data_asset_and_build_batch_request"
+```
+
+:::info Limit the Batches returned by a Batch Request
+
+You can provide a dictionary as the `options` parameter of `build_batch_request()` to limit the Batches returned by a Batch Request.  If you leave the `options` parameter empty, your Batch Request will include all the Batches configured in the corresponding Data Asset.  For more information about Batch Requests, see [How to request data from a Data Asset](/docs/guides/connecting_to_your_data/fluent/batch_requests/how_to_request_data_from_a_data_asset).
+
+:::
```

Comment:
Since we are not using the batch to create an expectation suite, would it make sense to remove this section (and re-number the rest)?

Reply:
Ohhhhh I understand now. Yes, you're right. I think I had modeled this a bit "too" closely with the validator-based guide :)